Bulls_Cows.py

```diff
@@ -0,0 +1,70 @@
+import random
+import time
+
+
+def generate_digits():
+    digits = []
+    while len(digits) < 4:
+        random_digit = random.randint(0,9)
+        if random_digit not in digits:
+            digits.append(random_digit)
+    return digits
+
+
+def ask_number():
+    print("Enter a number")
+    num = input()
+    return (list(int(i) for i in num))
```

Comment:
```
return [int(i) for i in number]
```
You should check that the user inputs number (not some character nonsense) and it has the right length.